### PR TITLE
Edits to Kathryn's OSS champions blog

### PR DESCRIPTION
**Pull request by kathrynkodama**
Fixes https://github.com/OpenLiberty/blogs/issues/1835

### CODE REVIEW

**Review by gljrobinson — DISMISSED** [review]
LGTM

**Comment by gljrobinson** [comment]
@dmuelle FYI

File: posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc [hunk 1]
```diff
@@ -34,11 +34,11 @@ Recently, Kathryn has been working on and contributing to the following projects
...
 
-The Open Liberty Tools for link:https://github.com/OpenLiberty/open-liberty-tools-vscode[VS Code] and link:https://github.com/OpenLiberty/open-liberty-tools-intellij[IntelliJ] are currently available as a tech preview and offer additional in-editor support for your Open Liberty projects. Both extensions contribute commands to easily start and stop Open Liberty dev mode, as well as run tests and view test reports. 
+The Open Liberty Tools extensions for link:https://github.com/OpenLiberty/open-liberty-tools-vscode[VS Code] and link:https://github.com/OpenLiberty/open-liberty-tools-intellij[IntelliJ] are currently available as a tech previews and offer additional in-editor support for your Open Liberty projects. Both extensions contribute commands to easily start and stop Open Liberty dev mode, as well as run tests and view test reports.
```
Comment (yeekangc) [inline posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc]:
```suggestion
The Open Liberty Tools extensions for link:https://github.com/OpenLiberty/open-liberty-tools-vscode[VS Code] and link:https://github.com/OpenLiberty/open-liberty-tools-intellij[IntelliJ] are currently available as tech previews and offer additional in-editor support for your Open Liberty projects. Both extensions contribute commands to easily start and stop Open Liberty dev mode, as well as run tests and view test reports. 
```

File: posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc [hunk 2]
```diff
@@ -65,15 +65,15 @@ image::/img/blog/KathrynKodama-2.png[Kathryn presenting at a virtual conference,
...
+IDE plugins, specifically the ones our team works on for MicroProfile and Open Liberty, offer our users an improved experience within their editors. From within their IDE, they are able to use tools like the MicroProfile Starter or Open Liberty dev mode, without having to reference external documentation or visit an external site. This enables developers to focus on the actual code of their application, as opposed to some of the details around initially creating a project, configuring Open Liberty, and deploying to a server. The Liberty developer experience team supports a number of editor plugins, such as the MicroProfile Starter for VS Code, the MicroProfile REST Client Generator for VS Code, and Open Liberty Tools for VS Code and IntelliJ (tech previews).
```
Comment (yeekangc) [inline posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc]:
```suggestion
IDE plugins, specifically the ones our team works on for MicroProfile and Open Liberty, offer our users an improved experience within their editors. From within their IDE, they are able to use tools like the MicroProfile Starter or Open Liberty dev mode, without having to reference external documentation or visit an external site. This enables developers to focus on the actual code of their application, as opposed to some of the details around initially creating a project, configuring Open Liberty, and deploying to a server. The Liberty developer experience team supports a number of editor plugins, such as the MicroProfile Starter for VS Code, the Generator for MicroProfile REST Client in VS Code, and Open Liberty Tools for VS Code and IntelliJ. 
```

File: posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc [hunk 3]
```diff
@@ -65,15 +65,15 @@ image::/img/blog/KathrynKodama-2.png[Kathryn presenting at a virtual conference,
...
 
 [#contributions]
 === In addition to contributing to code within these open source projects, what other contributions have you made? (i.e. documentation, tutorials, blogs, etc)
-I have contributed to the documentation on our IDE extensions; Open Liberty Tools for IntelliJ and VS Code, MicroProfile Starter for VS Code, the MicroProfile Extension Pack, etc. I have also created blog posts about our Open Liberty Tools for IntelliJ initial tech preview release, and the MicroProfile Language Server. Beyond these forms of communication, I have spoken at a number of presentations and conferences with the team. 
+I have contributed to the documentation on our IDE extensions; Open Liberty Tools for IntelliJ and VS Code, MicroProfile Starter for VS Code, the VS Code Extension Pack for MicroProfile, etc. I have also created blog posts about our Open Liberty Tools for IntelliJ initial tech preview release, and the Language Server for MicroProfile (Eclipse LSP4MP). Beyond these forms of communication, I have spoken at a number of presentations and conferences with the team.
```
Comment (yeekangc) [inline posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc]:
```suggestion
I have contributed to the documentation on our IDE extensions; Open Liberty Tools for IntelliJ and VS Code, MicroProfile Starter for VS Code, the VS Code Extension Pack for MicroProfile (Tools for MicroProfile), etc. I have also created blog posts about our Open Liberty Tools for IntelliJ initial tech preview release, and the Language Server for MicroProfile (Eclipse LSP4MP). Beyond these forms of communication, I have spoken at a number of presentations and conferences with the team. 
```

**Review by yeekangc — DISMISSED** [review]

**Review by dmuelle — APPROVED** [review]

**Pull request merged**